akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -109,7 +109,7 @@ There are various ways to wire up different parts of a stream, the following exa
 
 Back-pressure explained
 -----------------------
-Akka Streams implements an asynchronous non-blocking back-pressure protocol standardised by the `Reactive Streams`_
+Akka Streams implement an asynchronous non-blocking back-pressure protocol standardised by the `Reactive Streams`_
```

Comment:
In all the other chapters its used in plural. I suggest we make it consistent first and change it to singular later if necessary.